test/parallel/test-tls-client-mindhsize.js

```diff
@@ -75,6 +75,20 @@ function testDHE2048() {
 
 testDHE1024();
 
+assert.throws(() => test(512, true, assert.fail),
+              /DH parameter is less than 1024 bits/);
+
+assert.throws(() => tls.connect({ minDHSize: 0 }),
+              /minDHSize is not a positive number/);
+
+assert.throws(() => tls.connect({ minDHSize: -1 }),
+              /minDHSize is not a positive number/);
+
+[true, false, null, undefined, {}, [], '', '1'].forEach(value => {
```

Comment:
I added tests for negative infinity and nan.  I left out positive infinity because I'm not even sure what it would mean to want keys of infinite size.